coderd/workspaceagents.go

```diff
@@ -1524,7 +1536,9 @@ func (api *API) workspaceAgentPostMetadata(rw http.ResponseWriter, r *http.Reque
 	key := chi.URLParam(r, "key")
 
 	const (
-		maxValueLen = 32 << 10
+		// maxValueLen is set to 4096 to stay under the 8000 byte Postgres
+		// NOTIFY limit.
+		maxValueLen = 4096
```

Comment:
Can there be both a value and an error, resulting in 2*4096?

Reply:
ah yeah